vitals/BatteryFactorAccumulator.java

```diff
@@ -0,0 +1,89 @@
+package vitals;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BatteryFactorAccumulator {
+	
+	public static String language;
+	static List<String> lang_types = Arrays.asList("EN", "DE");
+	static Map<String,Float> VitalFactorExceedMinLimit=new HashMap<>();
+	static Map<String,Float> VitalFactorReachHighBreach=new HashMap<>();
+	static Map<String,Float> VitalFactorReachLowBreach=new HashMap<>();
+	static Map<String,Float> VitalFactorExceedMaxLimit=new HashMap<>();
+	
+
+    public static void showMsg(String msg)
+    {
+      System.out.println(msg);
+    }
+    
+    public static void displayMaximumAlert(String vitalparam)
+    {
+    	String alert = (language == "EN")? vitalparam + " is more than its Maximum Limit":"Die " + vitalparam + "ist hoher als die maximale Grenze";

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [ES: Comparison of String objects using == or != (ES_COMPARING_STRINGS_WITH_EQ)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq).
This code compares java.lang.String objects for reference equality using the == or != operators. Unless both strings are either constants in a source file, or have been interned using the String.intern() method, the same string value may be represented by two different String objects. Consider using the equals(Object) method instead.